Test Automation Course./src/topic6/ContulPersoanei.java

```diff
@@ -0,0 +1,63 @@
+package topic6;
+
+public class ContulPersoanei {
+    private String numePersoana;
+    private String id;
+    private double sumaCurenta;
+
+    public ContulPersoanei(String numePersoana, String id, double sumaCurenta) {
+        this.numePersoana = numePersoana;
+        this.id = id;
+        this.sumaCurenta = sumaCurenta;
+    }
+
+    public String getNumePersoana() {
+        return numePersoana;
+    }
+
+    public void setNumePersoana(String numePersoana) {
+        this.numePersoana = numePersoana;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public double getSumaCurenta() {
+        return sumaCurenta;
+    }
+
+    public void setSumaCurenta(double sumaCurenta) {
+        this.sumaCurenta = sumaCurenta;
+    }
+
+    public void sustragereDeBani(double suma) {
+        if (sumaCurenta > suma && suma > 0)
+            this.sumaCurenta = sumaCurenta - suma;
+        else {
+            throw new IllegalArgumentException("suma introdusa este gresita");
+
+        }
+
+    }
+
+    public void transferCatrePersoana(double suma, ContulPersoanei persoana) {
+        try {
+            if (suma > 0 && suma <= sumaCurenta) {
+                this.sumaCurenta -= suma;
+                persoana.sumaCurenta += suma;
+            } else {
+                throw new IllegalArgumentException("Suma solicitata e mai mare ca cea curenta");
```

Comment:
exceptiile nu se arunca ca sa fie prinse tot mai jos, ci in clasa care deja utilizeaza aceasta metoda